src/pages/categories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import NavBar from '../components/header/navigation';
+
+const Categories = () => (
+  <div className="container">
+    <NavBar />
+  </div>
```

Comment:
### 🛑 Required Changes
- [ ] Categories component is meant to display a button with text "Check status". Kindly fix this issue.